SalesforceCommerceCloud/pwa-kit
# @W-16649600 E2E Browser Tests for Do Not Track UI

# Description

These are the happy path E2E browser tests for the DNT UI in PWA Kit. 

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1) Added E2E Tests
- (change2) Set defaultDnt for the app and added a constant `DEFAULT_DNT_STATE` to constants.js

# How to Test-Drive This PR

- (step1) Pull this branch locally
- (step2) `export RETAIL_APP_HOME=http://localhost:3000`
- (step3) `npm run test:e2e:ui` and run the tests for dnt.spec.js
- (step4) See all tests pass

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
